packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -5,24 +5,55 @@ category: rate limits
+
+* [`system.client.concurrency_rate_limit.violation`](/docs/reference/api/event-types/?q=system.client.concurrency_rate_limit.violation)<br>
+This event is fired when the framework is in **Enforce and log per client** mode and a specific client, IP address, or device identifier combination makes more than two concurrent requests. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
 
-Additionally, there are specific [org-based System Log events](/docs/reference/api/system-log/#system-events) and [client-based System Log events](/docs/reference/rl-clientbased/#system-log-events). The org-based System Log events record system events related to your organization to provide an audit trail that you can use to understand platform activity and to diagnose problems. Client-based System Log events are fired when an individual client exceeds its assigned limit for the OAuth `/authorize` endpoint. Which event fired depends on the client-based rate-limiting mode that is set.
+* [`system.client.rate_limit.notification`](/docs/reference/api/event-types/?q=system.client.rate_limit.notification)<br>
+This event is fired when the framework is in **Log per client** mode and a specific client, IP address, or device identifier combination exceeds the total limit of 60 requests per minute. However, the end user won't see a rate-limit violation. Okta fires only a `notification` System Log event. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* [`system.client.concurrency_rate_limit.notification`](/docs/reference/api/event-types/?q=system.client.concurrency_rate_limit.notification)<br>
+This event is fired when the framework is turned on in **Log per client** mode and a specific client, IP address, Device token combination makes more than two concurrent requests. However, the end user won't see a rate-limit violation. Okta fires only a `notification` System Log event. The System Log contains information about the client ID, IP address, Device identifier, and the actual user if the user already has a valid session.
+
+### Operation rate limits
+
+Some [rate limits](/docs/reference/rate-limits/) are enforced on specific actions within Okta, regardless of which API is called to invoke the action. For example, though there are multiple ways to initiate an SMS to a user, there may be a limit on how many are sent out, regardless of which API requests has been made to initiate the messages. The following event types may appear in these varying cases:
```

Comment:
I don't think this link really makes sense here.

Reply:
Removed the link.